Source/RP0/Programs/Program.cs

```diff
@@ -569,5 +558,36 @@ public void SetBestAllowableSpeed()
         }
 
         public ProgramStrategy GetStrategy() => StrategySystem.Instance.Strategies.Find(s => s.Config.Name == name) as ProgramStrategy;
+
+        /// <summary>
+        /// Applies all active program modifiers to current program.
+        /// This should NOT get called on template programs (i.e the ones in ProgramHandler.Programs) since there is currently no logic that resets them to original state.
+        /// </summary>
+        public void ApplyProgramModifiers()
+        {
+            var pms = ProgramHandler.ProgramModifiers.FindAll(pm => pm.tgtProgram == name);
+            foreach (ProgramModifier pm in pms)
+            {
+                if (ProgramHandler.Instance.IsProgramActiveOrCompleted(pm.srcProgram))
+                {
+                    pm.Apply(this);
```

Comment:
What if you define two PM config with different source programs that modifies the same target program, you'd apply both and I am not sure the result would be deterministic here or be the intended effect?

Reply:
PMs get applied in the order they are configured and thus the effects should be deterministic.

Reply:
So since "modifiers" just a new value configuration, the last one configured is the only one used at the end right? or could we do partial PM configs that could be combined? (One defines nee confidence, one defines new fundingcurve)

Reply:
They would combine.